genie-web/build.gradle

```diff
@@ -117,6 +117,8 @@ node {
 }
 
 task bundle(type: NpmTask) {
+    inputs.files(fileTree("${project.projectDir}/src/main/resources/static"))
```

Comment:
It also seems like you should add two `inputs.file` lines and add the `webpack.config.js` and `package.json` in case there are changes to them.